CONTRIBUTING.md

```diff
@@ -74,9 +74,11 @@ Run a single test
 `$ rake test TEST=path/to/test.rb TESTOPTS="--name=test_something"`
 
 Run tests against different Rails versions by setting the RAILS_VERSION variable
-and bundling gems.
+and bundling gems.  (save this script somewhere executable and run from top of AMS repository)
```

Comment:
wwtd sounds like a great idea. Also, pulling the versions from .travis.yml would be very nice.